.github/workflows/test-build.yml

```diff
@@ -26,15 +27,15 @@ jobs:
           go-version: ${{ matrix.go-version }}
       - name: Display Go Version
         run: go version
-      - name: mod-check
+      - name: Run Go mod check
         run: make mod-check
-      - name: lint
+      - name: Run lint
         run: make lint
-      - name: test
+      - name: Run test
         run: make test
-      - name: test-benchmarks
+      - name: Run test benchmarks
         run: make test-benchmarks
-      - name: check-protos
+      - name: Run check protos and generate Go code from the proto files
```

Comment:
I'm not sure "...and generate Go code from the proto files" is accurate - doesn't this target just check that the generated files are up-to-date?

Reply:
> doesn't this target just check that the generated files are up-to-date?

Yes, but it will run also the `clean-protos` and `protos` target if they are not up to date. That's why I put the initial name for the this step with that description. However, at the end maybe I will simplify it to Check protos